static/lang/re-en.json

```diff
@@ -2826,6 +2826,18 @@
                 "WeaponOrUnarmed": "Select a weapon or unarmed strike."
             },
             "Psychic": {
+                "PsychicRapport": {
+                    "Description": "After you @UUID[Compendium.pf2e.actionspf2e.Item.aBQ8ajvEBByv45yz]{Cast a Spell} with the mental trait, until the start of your next turn, you gain a +1 circumstance bonus to Deception, Diplomacy, and Intimidation checks, and to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.1xRFPTFtWtGJ9ELw]{Sense Motive}."
```

Comment:
```suggestion
                    "Description": "After you Cast a Spell with the mental trait, until the start of your next turn, you gain a +1 circumstance bonus to Deception, Diplomacy, and Intimidation checks, and to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.1xRFPTFtWtGJ9ELw]{Sense Motive}."
```

Reply:
Any reason it's preferred to not link the action here? Either way, I've done it inconsistently here so I have to commit to one

Reply:
I think it's considered a common action, so typically it isn't linked. It also seems unlikely that the Cast a Spell link will be used in the context of a roll note, so it just makes it cluttered.

Reply:
Fair enough, I'll go without links for now just to fix my inconsistency. If we want them back Tikael can let me know.